htdocs/api/v0.0.3-dev/.htaccess

```diff
@@ -32,7 +32,7 @@ RewriteRule ^candidates/([0-9]+)/([a-zA-Z0-9_.]+)/instruments/([a-zA-Z0-9_.]+)/d
 ## Imaging API related URLs
 RewriteRule ^candidates/([0-9]+)/([a-zA-Z0-9_.]+)/images$ candidates/visits/Images.php?CandID=$1&VisitLabel=$2&PrintImages=true [L]
 RewriteRule ^candidates/([0-9]+)/([a-zA-Z0-9_.]+)/qc/imaging$ candidates/visits/qc/Imaging.php?CandID=$1&VisitLabel=$2&PrintQC=true [L]
-RewriteRule ^candidates/([0-9]+)/([a-zA-Z0-9_.]+)/images/([a-zA-Z0-9_.]+)/qc$ candidates/visits/images/qc/Qc.php?CandID=$1&VisitLabel=$2&Filename=$3&PrintImageQC=true [L]
+RewriteRule ^candidates/([0-9]+)/([a-zA-Z0-9_.]+)/images/([a-zA-Z0-9_.]+)/qc$ candidates/visits/images/qc/QC.php?CandID=$1&VisitLabel=$2&Filename=$3&PrintImageQC=true [L]
```

Comment:
on another unrelated note, shouldn't we allow a hyphen in the filenames? @driusan @xlecours 
currently as I try to download something from the test.loris.ca's API, it won;t work because the MRI project name is `loris-mri`.

Reply:
can I then fix it in this pull request, or better to issue another one?